src/Giraffe/EndpointRouting.fs

```diff
@@ -267,6 +285,7 @@ module Routers =
         | NestedEndpoint(t, lst, ce) -> NestedEndpoint(t, lst, ce >> f)
         | MultiEndpoint(lst) -> MultiEndpoint(List.map (configureEndpoint f) lst)
 
+    // XXX should we keep this?
```

Comment:
Is it used anywhere?

Reply:
I don't think so, but since I'm not sure why it was added in the first place, I'll leave it there and remove the comment.

Reply:
Removed this comment